Corona-Warn-App/src/main/res/values-de/event_registration_strings.xml

```diff
@@ -98,4 +98,11 @@
     <!-- XTXT: Title for other event  -->
     <string name="tracelocation_organizer_category_other_event_title">anderes Event</string>
 
+
+    <!--    Event Registration Homescreen Card-->
+    <!-- XHED: Headline for event registration creation card -->
+    <string name="create_event_card_headline">"Sie planen eine\nVeranstaltung?"</string>
+    <string name="create_event_card_subtitle">"Sie planen ein Event oder haben ein Geschäft? Erstellen Sie einen QR-Code, mit dem sich Ihre Gäste bei Ankunft einchecken können."</string>
+    <string name="create_event_card_button">"QR-Code erstellen"</string>
```

Comment:
*_trace_locaion_* ?

Reply:
@AlexanderAlferov  did you already used `trace_location_` prefix for the strings , should those strings follow the same pattern?